sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items-mlflow.ipynb

```diff
@@ -1160,7 +1161,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.10"
+   "version": "3.8.12"
```

Comment:
is this change intentional?

Reply:
Yes.  It is not very significant, but we do test on Python 3.8